src/components/Reels.jsx

```diff
@@ -29,20 +28,20 @@ const StyledReel = styled.div`
   border-radius: 10px;
 `;
 
-
 export const Reels = (props) => {
   const { reelValues } = props;
-  console.log('reelValues', reelValues);
+  console.log("reelValues", reelValues);
```

Comment:
we usually clean this up and remove before merging the feature in. Remove this log.